netbox/dcim/forms/bulk_import.py

```diff
@@ -471,7 +471,7 @@ class Meta:
         model = ModuleType
         fields = [
             'manufacturer', 'model', 'part_number', 'description', 'airflow', 'weight', 'weight_unit', 'comments',
-            'tags',
+            'tags', 'profile',
```

Comment:
Could you move `profile` before `comments` in the list? It affects the ordering of the fields in the import form.